src/apps/vpn/models/device.cpp

```diff
@@ -42,7 +42,7 @@ QString Device::currentDeviceName() {
       // status.
       MacOSUtils::computerName();
 #elif MZ_ANDROID
-      AndroidUtils::GetDeviceName();
+      AndroidUtils::getDeviceModel();
```

Comment:
The name of this method is inconistent with the corresponding names in the other utils. To make this consistent should AndroidUtils::getDeviceModel be renamed to AndroidUtils::getDeviceName, and name the getDeviceName to getDeviceCodename, because "Panther" etc. are code names.